module/documents/actor/actor.mjs

```diff
@@ -124,6 +124,15 @@ export default class Actor5e extends Actor {
 
   /* --------------------------------------------- */
 
+  /** @override */
+  prepareEmbeddedDocuments() {
+    super.prepareEmbeddedDocuments();
```

Comment:
This call to `super.prepareEmbeddedDocuments` (in this case `Actor#prepareEmbeddedDocuments`) will also call `this.applyActiveEffects`, no? I suppose we could move `this._prepareScaleValues` to the top of the `applyActiveEffects` override instead?